### PR TITLE
Identity lookup

### PR DESCRIPTION
**Pull request by rogeralsing**
continuing @umitkavala's work in this branch

### CODE REVIEW

File: cluster/member.go [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+
+	//I assume this is not the fastest way to do this?
+	s := ""
+	for _, m := range members {
+		s += m.Id
+	}
+
+	//TODO: this HAS to be compatible with the same hash in .NET
+	//add plenty of tests
+	hash := murmur32.Sum32([]byte(s))
```
Comment (avinash-getmega) [inline cluster/member.go]:
```suggestion
	var s strings.Builder
	s.Grow(len(members)*LENGTH_OF_EACH_MEMBER_ID)
	for _, m := range members {
		fmt.Fprintf(&s, "%s", m.Id)
	}
	
	//TODO: this HAS to be compatible with the same hash in .NET
	//add plenty of tests
	hash := murmur32.Sum32([]byte(s.String()))
```

Comment (avinash-getmega) [reply cluster/member.go]: `strings` package needs to be added to the file and `LENGTH_OF_EACH_MEMBER_ID` needs to be defined or used here if already defined somewhere

File: actor/config.go [hunk 2]
```diff
@@ -22,31 +22,47 @@ func defaultActorSystemConfig() Config {
 	}
 }
 
-func NewConfig() Config {
-	return defaultActorSystemConfig()
+type ConfigOption func(config Config) Config
```
Comment (chongkai) [inline actor/config.go]: if it's func(*Config), then we don't need to return the value, avoiding copying values in and out
`type ConfigOption func(*Config)`

**Comment by rogeralsing** [comment]
@all-contributors please add @rogeralsing for code

**Comment by rogeralsing** [comment]
@all-contributors please add @DamnWidget for code

**Comment by rogeralsing** [comment]
@all-contributors please add @umitkavala for code

**Comment by rogeralsing** [comment]
@all-contributors please add @umitkavala for code

**Comment by rogeralsing** [comment]
Hopefully the last error to fix to get it all working again

```
github.com/AsynkronIT/protoactor-go/cluster.(*Rendezvous).GetByClusterIdentity(0x0?, 0x103fd2538?)
	/Users/rogerjohansson/git/asynkron/protoactor-go/cluster/rendezvous.go:30 +0x30
github.com/AsynkronIT/protoactor-go/cluster/identitylookup/disthash.(*Manager).Get(0x140002b7280, 0x1400009ed20)
	/Users/rogerjohansson/git/asynkron/protoactor-go/cluster/identitylookup/disthash/manager.go:71 +0x30
github.com/AsynkronIT/protoactor-go/cluster/identitylookup/disthash.(*IdentityLookup).Get(0x1037f17a0?, 0x140001b0008?)
	/Users/rogerjohansson/git/asynkron/protoactor-go/cluster/identitylookup/disthash/identity_lookup.go:13 +0x24
github.com/AsynkronIT/protoactor-go/cluster.(*Cluster).Get(...)
	/Users/rogerjohansson/git/asynkron/protoactor-go/cluster/cluster.go:143
main.main()
	/Users/rogerjohansson/git/asynkron/protoactor-go/_examples/cluster-basic/node1/main.go:18 +0xc4
exit status 2
```

**Pull request merged**